packages/ember-views/lib/system/jquery_event_deprecation.js

```diff
@@ -33,6 +33,7 @@ export default function addJQueryEventDeprecation(jqEvent) {
             {
               id: 'ember-views.event-dispatcher.jquery-event',
               until: '4.0.0',
+              url: 'https://emberjs.com/deprecations/v3.x#toc_jquery-event'
```

Comment:
CI is failing on this line, needs a tailing comma I think.

You could also do `yarn lint:fix` I believe...

Reply:
Oh well, didn't expect this little change to fail. Should probably setup prettier in my IDE I guess... Fixed now! 😬